test/kubemark/common.sh

```diff
@@ -23,3 +24,27 @@ export PROJECT
 MASTER_NAME="${INSTANCE_PREFIX}-kubemark-master"
 MASTER_TAG="kubemark-master"
 EVENT_STORE_NAME="${INSTANCE_PREFIX}-event-store"
+
+RETRIES=3
+
+# arguments:
+# $1: name of operation to embed in an error message
+# $2...: all stuff that goes after 'gcloud compute '
+function run-gcloud-compute {
```

Comment:
do you want to move this to some place more usable from other setup scripts? (I assume they all have similar retry problems)